src/component/chat.cpp

```diff
@@ -21,6 +23,8 @@ namespace chat
 		std::vector<scripting::function> say_callbacks;
 		utils::hook::detour g_say_hook;
 
+		std::map<std::string, std::string> colors_users;
```

Comment:
I just realized all of this is pretty pointless since you have already the userinfo_overrides.
You should use those when overidding the chat probably

Reply:
So overwrite the value where is defined the team color and replace it with the correct one. I sthat what you mean?

Reply:
The dll provides the gsc functions setname/rename and clantag. You should use those names. Perhaps feds knows what he wants to do with this

Reply:
I'll see but if i remeber right add the color on the name value in the msg or change the name should not work. I'll try it